index.html

```diff
@@ -8,50 +8,266 @@
+                </p>
+                <div>
+                    <span class="skils">HTML</span>
+                    <span class="skils">Ruby on rails</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
+                <div class="button-container">
+                    <p class="button">See Project</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="card">
+        <div class="card-items">
+            <img src="./assets/gym-fit-bg.png" alt="" />
+            <div class="card-continent">
+                <div>
+                    <h1 class="card-header">Tonic</h1>
+                </div>
+                <div class="cards-elemnts">
+                    <span class="span-weight-500">CANBOY</span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">Back End Dev </span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">2018</span>
+                </div>
+                <p class="card-text">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div>
+                    <span class="skils">HTML</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
+                <div class="button-container">
+                    <p class="button">See Project</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="card">
+        <div class="card-items">
+            <img src="./assets/art-mobile-bg.png" alt="" />
+            <div class="card-continent">
+                <div>
+                    <h1 class="card-header">Multi-Post Stories
+                    </h1>
+                </div>
+                <div class="cards-elemnts">
+                    <span class="span-weight-500">CANBOY</span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">Back End Dev </span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">2018</span>
+                </div>
+                <p class="card-text">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div>
+                    <span class="skils">HTML</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
+                <div class="button-container">
+                    <p class="button">See Project</p>
```

Comment:
- [ ] This element is meant to be a button. Please change the `<p>` tag to a link `<a>` or `<button>` tag, and follow the button styling: Active, Hover, and Pressed.
![Capture](https://user-images.githubusercontent.com/85095702/159127556-1db3c7bd-5d8a-4a13-8f7a-e80366acad3a.PNG)